API/Handlers/Implementations/OAuthHandler.cs

```diff
@@ -57,6 +61,34 @@ IPasswordHasher<OAuthClient> clientSecretHasher
         };
     }
 
+    /// <summary>
+    /// Sync the user's osu! friends list with the repository
+    /// </summary>
+    /// <param name="user">The user</param>
+    private async Task SyncFriendsAsync(User user)
+    {
+        // Do not allow friends list syncs more often than every 24 hours
+        if (user.LastFriendsListUpdate is not null &&
+            (DateTime.UtcNow - user.LastFriendsListUpdate).Value < TimeSpan.FromDays(1))
```

Comment:
Would rather this timespan be configurable, or made a constant field of the class